README.md

```diff
@@ -191,6 +187,18 @@ It is possible to run gProfiler without using `perf` - this is useful where `per
 
 In this mode, gProfiler uses runtime-specific profilers only, and their results are concatenated (instead of scaled into the results collected by `perf`). This means that, although the results from different profilers are viewed on the same graph, they are not necessarily of the same scale: so you can compare the samples count of Java to Java, but not Java to Python.
 
+# Building
+
+## Container
+
+* x86_64: `./scripts/build_x86_64_container.sh -t gprofiler` will create a local image `gprofiler`.
+* Aarch64: `./scripts/build_x86_64_container.sh`, you will need to set up buildx for building cross architecture before, if you're building on x86_64.
```

Comment:
Script name seems off if it's used for all archs

Reply:
Uhh it's a typo, should be `aarch64`

Reply:
@Jongy  fix typo